packages/react/src/components/MultiSelect/MultiSelect.js

```diff
@@ -447,6 +461,7 @@ MultiSelect.defaultProps = {
   compareItems: defaultCompareItems,
   disabled: false,
   locale: 'en',
+  itemToElement: null,
```

Comment:
```suggestion
```

I think we could leave it undefined with the checks above before calling, what do you think?

Reply:
Definitely 👍🏻